cohorts/utils.py

```diff
@@ -16,6 +16,22 @@
 
 import re
 import warnings
+from collections import namedtuple
+
+class DataFrameHolder(namedtuple("DataFrameHolder", ["cols", "df"])):
+    """Holds a DataFrame along with associated columns of interest."""
+    def return_self(self, return_cols):
+        """Either return the DataFrame or the DataFrameHolder."""
+        if return_cols:
+            return self
+        else:
+            return self.df
+
+    @staticmethod
+    def return_obj(cols, df, return_cols=False):
```

Comment:
I still use it as a `tuple` in a bunch of places; made for a less intensive refactor. Does that seem reasonable?